nuget.config

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<packageSources>
+		<clear />
+  		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+	</packageSources>
+	<packageSourceMapping>
+  		<packageSource key="nuget.org">
+    			<package pattern="*" />
+  		</packageSource>
+	</packageSourceMapping>
```

Comment:
identation very strange

Reply:
Well when you copy paste from discord to an IDE that is what happens

Reply:
idk its whatever neovim decided, ill reformat it to 2 spaces which seems to be the standard for XML?

Reply:
> Well when you copy paste from discord to an IDE that is what happens

that was not the issue